resources/tests/test_resources_queries.py

```diff
@@ -10,6 +10,18 @@
 from leases.tests.factories import BerthLeaseFactory
 
 
+@pytest.mark.parametrize(
+    "api_client",
+    [
+        "api_client",
+        "user",
+        "harbor_services",
+        "berth_supervisor",
+        "berth_handler",
+        "berth_services",
+    ],
+    indirect=True,
+)
```

Comment:
I think it's enough if we use the non-parametrized `api_client` with the `AnonymousUser` in it 🙂